### PR TITLE
Migrate to hermesc

### PR DESCRIPTION
**Pull request by nikosszzz**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { randomBytes } from "crypto";
 
 export function hermes(options?: { hermesPath?: string }): Plugin {
     options ??= {};
-    options.hermesPath ??= join(__dirname, "..", "..", "hermes-engine"); // node_modules/rollup-plugin-hermes/dist/../../hermes-engine
+    options.hermesPath ??= join(__dirname, "..", "..", "@aliucord/hermes"); // node_modules/rollup-plugin-hermes/dist/../../@aliucord/hermes
```
Comment (rushiiMachine) [inline src/index.ts]:
```suggestion
    options.hermesPath ??= join(__dirname, "../../@aliucord/hermes"); // node_modules/@aliucord/hermes
```

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/Aliucord/rollup-plugin-hermes#readme",
   "peerDependencies": {
-    "hermes-engine": "0.x"
+    "@aliucord/hermesc": "^0.70.6-aliucord-1"
```
Comment (rushiiMachine) [inline package.json]:
```suggestion
    "@aliucord/hermesc": "latest"
```

**Review by rushiiMachine — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (rushiiMachine) [inline package-lock.json]: guh hop on pnpm (update the lockfile)

**Review by rushiiMachine — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { randomBytes } from "crypto";
 
 export function hermes(options?: { hermesPath?: string }): Plugin {
     options ??= {};
-    options.hermesPath ??= join(__dirname, "..", "..", "hermes-engine"); // node_modules/rollup-plugin-hermes/dist/../../hermes-engine
+    options.hermesPath ??= join(__dirname, "../../@aliucord/hermes"); // node_modules/@aliucord/hermes
```
Comment (amsryq) [inline src/index.ts]: Just keep the context
```suggestion
    options.hermesPath ??= join(__dirname, "../../@aliucord/hermes"); // node_modules/rollup-plugin-hermes/dist/../../@aliucord/hermes
```

File: package-lock.json (hunk 3 above)
Comment (nikosszzz) [reply package-lock.json]: Guh i used pnpm and lockfile remained the same for whatever reason

Comment (nikosszzz) [reply package-lock.json]: it wont be in lockfile cause hermesc is peer dependency

**Pull request merged**